src/main/resources/com/arcbees/website/server/html/home.vm

```diff
@@ -42,22 +42,14 @@
         }
     </style>
 
+    <script>
```

Comment:
I added the analytics.js script, then the 2nd one is the Content Experiment API script, that we can use to get/set/choose a variant for the experiment